docs/_static/custom.css

```diff
@@ -25,3 +25,23 @@
 .toggle{
   background: #FBFBFB;
 }
+
```

Comment:
The css styles file here is common with Qiskit to keep the same style throughout the docs. I think we should keep it that way - if there some style improvement it should get done Qiskit in general I think.

Reply:
I opened [this issue](https://github.com/Qiskit/qiskit_sphinx_theme/issues/117) in the `qiskit_sphinx_theme` to fix the class and method header styles, but i don't know when it will get prioritised and it will still be a while before the next version of the theme is released and ready for all repos to use. The fix in this PR would work for the time being if you want the styles fixed in time for the nature release.

Reply:
So I guess it depends how soon you want the style fix? In this case I would advocate for changing it here (and also I can open PRs for a similar fix in some of the other repos maybe?) so at least the users get the benefit of the better styles straight away? But if you would rather wait lmk

Reply:
I would prefer to keep things consistent and wait for if/when the theme etc gets updated for all Qiskit so that its picked up everywhere, whatever is done, rather than have specifics here so its different.

Reply:
I am indifferent to this. Keeping things consistent might be easier down the road but this change seems so minor and yields much better results that I would also not see this as a hassle. Especially since this is aligned with the rest of the theme and really just fixing a bug by adding attributes that were missing before